source/_redirect/_tmp/redirect-1.blade.md

```diff
@@ -0,0 +1,4 @@
+---
+from: /account
```

Comment:
Will be necessary do an amend to remove the _redirect and _team folder

Reply:
removed

Reply:
Only by study suggestion.

At this case the best way is to use rebase and delete the files from git History. Without this, a good practice is to squash all commits to we don't have a lot of files added to history and after a lot of files removed, this action increase the history and make slow to handle the repository.